lib/modules/apostrophe-schemas/index.js

```diff
@@ -1619,6 +1619,27 @@ module.exports = {
       }
     });
 
+    self.addFieldType({
+      name: 'email',
+      converters: {
+        string: function (req, data, name, object, field, callback) {
+
+          if (field.required && (_.isUndefined(data[name]) || !data[name].toString().length)) {
+            return callback('required');
+          }
+
+          var matches = data[name].match(/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/);
```

Comment:
This... looks good? (: Can you add a link in a comment to the source where you found this regex? I think it'll be helpful to know where it came from if we need to revise it in the future.